diff --git a/node_modules/react-native-maps/lib/MapMarker.js b/node_modules/react-native-maps/lib/MapMarker.js
index 90b689e..d22177f 100644
--- a/node_modules/react-native-maps/lib/MapMarker.js
+++ b/node_modules/react-native-maps/lib/MapMarker.js
@@ -29,7 +29,7 @@ const react_native_1 = require("react-native");
 const decorateMapComponent_1 = __importStar(require("./decorateMapComponent"));
 const MapMarkerNativeComponent_1 = require("./MapMarkerNativeComponent");
 class MapMarker extends React.Component {
-    getNativeComponent;
+    // getNativeComponent;
     getMapManagerCommand;
     getUIManagerCommand;
     static Animated;
diff --git a/node_modules/react-native-maps/lib/MapPolygon.js b/node_modules/react-native-maps/lib/MapPolygon.js
index 86b8fee..0134fb6 100644
--- a/node_modules/react-native-maps/lib/MapPolygon.js
+++ b/node_modules/react-native-maps/lib/MapPolygon.js
@@ -27,7 +27,7 @@ exports.MapPolygon = void 0;
 const React = __importStar(require("react"));
 const decorateMapComponent_1 = __importStar(require("./decorateMapComponent"));
 class MapPolygon extends React.Component {
-    getNativeComponent;
+    // getNativeComponent;
     getMapManagerCommand;
     getUIManagerCommand;
     polygon;
diff --git a/node_modules/react-native-maps/lib/MapPolyline.js b/node_modules/react-native-maps/lib/MapPolyline.js
index b0efda7..260ff44 100644
--- a/node_modules/react-native-maps/lib/MapPolyline.js
+++ b/node_modules/react-native-maps/lib/MapPolyline.js
@@ -27,7 +27,7 @@ exports.MapPolyline = void 0;
 const React = __importStar(require("react"));
 const decorateMapComponent_1 = __importStar(require("./decorateMapComponent"));
 class MapPolyline extends React.Component {
-    getNativeComponent;
+    // getNativeComponent;
     getMapManagerCommand;
     getUIManagerCommand;
     polyline;
